src/docs/integrations/pagerduty.mdx

```diff
@@ -0,0 +1,41 @@
+---
+title: "PagerDuty Integration"
+---
+
+## Create a PagerDuty app
+1. In PagerDuty, in the top right, click on "Developer mode"
```

Comment:
This option is in a dropdown in the 3rd icon from the right (idk how to describe the 3 squares with a plus sign better than that), could we be more explicit? This dummy (me) was expecting it to be in the farthest, rightest option.

Reply:
Hmm yea that's a weird button to describe. I'm just going to give the URL.